### DISCUSSION TITLE
Rack location on site

### DISCUSSION THREAD

**Discussion by LensHunnel** [nautobot/nautobot#3281] [Plugin (App) Ideas & Development]
### Environment
* Python version: 3.7
* Nautobot version: 1.0.3

### Proposed Functionality

locate a rack inside a site's internal map. For example :   

![image](https://user-images.githubusercontent.com/6883486/130599309-19c44461-af87-475e-b2fc-2264c60a57ac.png)

### Use Case
In the event of a maintenance operation, it may be useful to be able to identify a rack on an site internal map

**Comment by jedelman8** [comment]
Are you proposing the idea to upload an existing diagram and overlay hyperlinks on it or the ability to have Nautobot dynamically create a rack diagram per site allowing users to "move racks" in the right floor plan?

**Comment by glennmatthews** [comment]
I think this would be terrific as a plugin -- it's not something we're able to invest time into at the present but it should be very doable as a standalone plugin by someone willing to put in the effort of developing it. :-)
